gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java

```diff
@@ -293,7 +293,7 @@ protected T getElement(Widget item) {
 
     @Override
     public Object[] getChildren(Object parent) {
-      return !hasChildren(parent) ? null : getChildNodes(cast(parent));
+      return !hasChildren(parent) ? new Object[0] : getChildNodes(cast(parent));
```

Comment:
Shouldn't calls to getChildren check for null instead of returning an empty array?

Reply:
`getChildren` is called from the JFace library, which is where the exception originates.